.github/workflows/yarn.yml

```
@@ -0,0 +1,22 @@
+# Workflow will do a fresh install and build of node using Yarn
+
+name: Yarn CI
+
+on:
+  push:
+    branches: [ development ]
```

```suggestion
    branches: [ development, staging, master ]
```

I think this can be a start, but ideally the workflow should run at every push also in the feature branches. We can improve this later with a new PR.

At least I would allow for the other main trunks to have their workflow so that the CI can be in a complete state in its first form.